src/main/java/com/springboot/star_wars_character_popularity/app/service/serviceImpl/VoteServiceImpl.java

```diff
@@ -60,4 +68,39 @@ public void deleteVote(long id) {
 
         voteRepository.deleteById(id);
     }
+
+    @Override
+    public Vote saveVote(Vote vote, MultipartFile multipartFile) throws IOException {
+       Vote dbVote = voteRepository.save(vote);
+
+       if(dbVote!=null && multipartFile!=null) {
+
+           MultipartFile file = multipartFile;
+
+           String fileName = file.getOriginalFilename();
+           String modifiedFileName = FilenameUtils.getBaseName(fileName) + System.currentTimeMillis() + FilenameUtils.getExtension(fileName);
+           File storeFile = uploadPathService.getFilePath(modifiedFileName, "images");
```

Comment:
remove saving file to disk, use FileUploadUtil.saveFile instead

Reply:
Done.